sdk/monitor/monitor-query/samples/v1/typescript/package.json

```diff
@@ -29,7 +29,7 @@
   "dependencies": {
     "@azure/monitor-query": "latest",
     "dotenv": "latest",
-    "@azure/identity": "^4.2.1"
+    "@azure/identity": "^4.0.1"
```

Comment:
Same here - why is this changed to older version?

Reply:
This is auto-generated